commit 711b46f8fc4e8a3d5caf7d4820562d6cdfe9d769
Author: Paul Eggert <eggert@cs.ucla.edu>
Date:   Tue Aug 9 07:02:19 2022 -0700

    Chile's DST is delayed by a week in September 2022
    
    * southamerica (Chile): Chile's 2022 start this spring is delayed
    from 09-04 to 09-11.  (Thanks to Juan Correa.)  For now, assume
    this is a one-off due to the vote on the new constitution.

--- a/NEWS
+++ b/NEWS
@@ -4,6 +4,9 @@
 
   Changes to future timestamps
 
+    Chile's 2022 DST start is delayed from September 2 to September 11.
+    (Thanks to Juan Correa.)
+
     Iran plans to stop observing DST permanently, after it falls back
     on 2022-09-21.  (Thanks to Ali Mirjamali.)
 
--- a/southamerica
+++ b/southamerica
@@ -1258,6 +1258,14 @@
 # For now, assume that they will not revert,
 # since they have extended the expiration date once already.
 
+# From Juan Correa (2022-08-09):
+# the Internal Affairs Ministry (Ministerio del Interior) informed DST
+# for America/Santiago will start on midnight of September 11th;
+# and will end on April 1st, 2023. Magallanes region (America/Punta_Arenas)
+# will keep UTC -3 "indefinitely"...  This is because on September 4th
+# we will have a voting whether to approve a new Constitution.
+# https://twitter.com/min_interior/status/1556981967551209472
+
 # Rule	NAME	FROM	TO	-	IN	ON	AT	SAVE	LETTER/S
 Rule	Chile	1927	1931	-	Sep	 1	0:00	1:00	-
 Rule	Chile	1928	1932	-	Apr	 1	0:00	0	-
@@ -1294,7 +1302,9 @@
 Rule	Chile	2016	2018	-	May	Sun>=9	3:00u	0	-
 Rule	Chile	2016	2018	-	Aug	Sun>=9	4:00u	1:00	-
 Rule	Chile	2019	max	-	Apr	Sun>=2	3:00u	0	-
-Rule	Chile	2019	max	-	Sep	Sun>=2	4:00u	1:00	-
+Rule	Chile	2019	2021	-	Sep	Sun>=2	4:00u	1:00	-
+Rule	Chile	2022	only	-	Sep	Sun>=9	4:00u	1:00	-
+Rule	Chile	2023	max	-	Sep	Sun>=2	4:00u	1:00	-
 # IATA SSIM anomalies: (1992-02) says 1992-03-14;
 # (1996-09) says 1998-03-08.  Ignore these.
 # Zone	NAME		STDOFF	RULES	FORMAT	[UNTIL]
